src/pages/contact_us/_offices.tsx

```diff
@@ -196,17 +220,62 @@ export const Offices = () => {
                     </ResHeader>
+                    <ContinentHeader>{localize('EUROPE')}</ContinentHeader>
                     <GridLayout>
+                        <Flex fd="column" max_width="48.6rem" id="france">
+                            <OfficeHeader>
+                                <div>
+                                    <img src={France} alt="France Icon" />
+                                </div>
+                                <Header as="h4" type="sub-section-title" mt="0.8rem" mb="1.6rem">
+                                    {localize('France')}
+                                </Header>
+                            </OfficeHeader>
+
+                            <BorderBox>
+                                <MapContainer>
+                                    <LocalizedLink
+                                        to="https://g.page/wojo-madeleine-coworking-paris-8?share"
+                                        external="true"
+                                        target="_blank"
+                                        rel="noopener noreferrer"
+                                    >
+                                        <QueryImage
+                                            alt="Deriv's office location map in France"
+                                            data={data['map_france']}
+                                            height="100%"
+                                        />
+                                    </LocalizedLink>
+                                </MapContainer>
+                                <LocalizedLinkText
+                                    to="https://g.page/wojo-madeleine-coworking-paris-8?share"
+                                    external="true"
+                                    target="_blank"
+                                    rel="noopener noreferrer"
+                                    mt="0.8rem"
+                                    ml="1.6rem"
+                                >
+                                    <Desktop>
+                                        <p>Wojo Madeleine, 18 Boulevard</p>
+                                        <p>Malesherbes, Paris 75008</p>
+                                    </Desktop>
+                                    <Mobile>
+                                        <StyledMobileAddress>
+                                            Wojo Madeleine, 18 Boulevard Malesherbes, Paris 75008
+                                        </StyledMobileAddress>
+                                    </Mobile>
+                                </LocalizedLinkText>
+                            </BorderBox>
+                        </Flex>
                         <Flex fd="column" max_width="48.6rem" id="malta">
                             <OfficeHeader>
                                 <div>
-                                    <img src={Malta} alt="" />
+                                    <img src={Malta} alt="Malta Icon" />
```

Comment:
The alt test does not look good
Please follow this PR's alt naming convention. 
https://github.com/binary-com/deriv-com/pull/3362/files